easybuild/easyconfigs/a/Amber/Amber-16-iomkl-2016.09-GCC-4.9.3-2.25-AmberTools-16-patchlevel-5-14-CUDA.eb

```diff
@@ -0,0 +1,36 @@
+# Automatically converted from Amber-16-iomkl-2016.09-CUDA.eb
+# by /gpfs/sysapps/bham/scripts/easybuild/recursive_build_with_new_toolchain.py
```

Comment:
@edmondac please remove

Remarks from 1st Amber easyconfig also apply to the others in this PR.